test/gun_papas_cfg.py

```diff
@@ -3,6 +3,8 @@
 import heppy.framework.config as cfg
 
 import logging
+
+import sys
```

Comment:
OK. Thanks

Have sorted some of these things already …  still tidying up a little before push  etc

A

> On 23 Feb 2016, at 17:55, Colin Bernet <notifications@github.com> wrote:
> 
> In test/gun_papas_cfg.py <https://github.com/cbernet/heppy/pull/2#discussion_r53810891>:
> 
> > @@ -3,6 +3,8 @@
> >  import heppy.framework.config as cfg
> >  
> >  import logging
> > +
> > +import sys
> can you remove this import if the sys module is not used? 
> if it's used in the main section only, the import should go there.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cbernet/heppy/pull/2/files#r53810891>.
>